src/main/java/at/hannibal2/skyhanni/utils/ItemUtils.kt

```diff
@@ -515,4 +515,9 @@ object ItemUtils {
             true, true,
         )
     }
+
+    fun getHeldItem(): ItemStack? {
```

Comment:
InventoryUtils.getItemInHand already exits

Reply:
figured something like this would already exist, ty